ruby-challenges.rb

```diff
@@ -32,3 +47,14 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+def palindrome(input)
+  if input.downcase == input.downcase.reverse
+    p "#{input} is a palindrome"
+  else
+    p "#{input} is not a palindrome"
```

Comment:
For all of these Ruby challenges, I would remove the `p` within your methods. These are essentially the same as leaving in console.logs in JavaScript.